DirectEmployers/MyJobs
# [PD-3470] fix reporting export - criteria sheet

## What's this PR do?
convert raw data from request to human readable format in the criteria sheet.

## Screenshots (if appropriate)
before:
![image](https://user-images.githubusercontent.com/27963853/28734407-d3ad9182-73ae-11e7-8ac9-f3c773fc5067.png)

after:
![image](https://user-images.githubusercontent.com/27963853/28734316-6322024a-73ae-11e7-8edc-b3a5a163c982.png)

## Where should the reviewer start?
`myreports/views.py` Line#750

## How should this be manually tested?
Go to PRM or TJD, pull any reports with different criteria and download the excel, check all criteria are represented correctly.

## Any background context you want to provide?
Added adorned filters as an additional argument to 2 functions, to help us translate some fields.
refactored the xlsx writer a bit.
